event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinMapper.java

```diff
@@ -63,11 +91,104 @@ public static DbCandidate toDbCandidate(CristinNviReport cristinNviReport) {
                    .basePoints(extractBasePoints(cristinNviReport))
                    .collaborationFactor(extractCollaborationFactor(cristinNviReport))
                    .internationalCollaboration(isInternationalCollaboration(cristinNviReport))
+                   .channelId(extractChannelId(cristinNviReport))
+                   .channelType(extractChannelType(cristinNviReport))
                    .build();
     }
 
+    public static String extractNode(JsonNode node, String jsonPointer) {
+        return attempt(() -> extractJsonNodeTextValue(node, jsonPointer)).orElse(failure -> null);
+    }
+
+    public static String getJsonNodeAt(JsonNode node, String jsonPointer) {
```

Comment:
Do you need this method? Looks like duplicate of `extractNode`-method?

Reply:
Yes, in case I am trying to extract non existent jsonNode, it throws exception. Migration data can have other publicationContext types and publicationInstance types than "our" nvi service allow.